guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p><em>DataFrame containing the dataset along with the y-</em><strong><em>vairable</em></strong><em> defined</em></p><blockquote>spellcheck <strong>vairable</strong> in the above line, in the last line of the paragraph</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>